src/components/editor/faultTree/Editor.tsx

```diff
@@ -44,6 +45,7 @@ const Editor = ({setAppBarName}: DashboardTitleProps) => {
 
     useEffect(() => {
         if (faultTree) {
+            console.log(faultTree.faultEventScenarios)
```

Comment:
```suggestion
```

Reply:
I do not consider it a good idea to include console logs.

Reply:
resolved